test/install_conda_packages_test.py

```diff
@@ -16,7 +16,13 @@ def main():
     parser.add_argument("docker_image", type=str, help="The docker image to use.")
 
     parser.add_argument("--python-package", default="katana-python", help="Name of python conda package to test")
+    parser.add_argument(
+        "--python-package-channels", default="-c katanagraph", help="Channels to use when installing the python package"
+    )
     parser.add_argument("--tools-package", default="katana-tools", help="Name of tools conda package to test")
+    parser.add_argument(
+        "--tools-package-channels", default="-c katanagraph", help="Channels to use when installing the tools package"
+    )
```

Comment:
I would really prefer these to be list arguments. Including `-c` in each argument is pretty weird.